packages/backend/src/models/ContentModel/ContentConfigurations/DashboardContentConfiguration.ts

```diff
@@ -67,7 +68,10 @@ export const dashboardContentConfiguration: ContentConfiguration<SummaryContentR
                     `last_version.updated_by_user_uuid`,
                 )
                 .select<SummaryContentRow[]>([
-                    knex.raw(`'dashboard' as content_type`),
+                    knex.raw(`'${ContentType.DASHBOARD}' as content_type`),
```

Comment:
it does not have to be in the select. you can just add it below in the return

Reply:
The return is for a converted row @IrakliJani, so it won't be possible to be used in a db query (in `ContentModel`)